indico/web/client/js/react/util/Overridable.jsx

```diff
@@ -18,10 +18,10 @@ import {withRouter} from 'react-router-dom';
  * @param {Object|Function} extraProps - additional properties passed to the final component
  */
 export function parametrize(Component, extraProps) {
-  const ParametrizedComponent = ({...props}) => {
```

Comment:
This looks like it was meant to explicitly copy the props (or maybe it was just a leftover from when we had a non-...rest prop as well?)

Reply:
I didn't find any reason to actually copy the properties over there. That's why I removed it.